composer/datasets/utils.py

```diff
@@ -15,6 +15,9 @@
 from torchvision.datasets import VisionDataset
 
 from composer.core import Batch
+import transformers
+if TYPE_CHECKING:
+    import transformers
```

Comment:
Hmm this looks strange, if we need to import `transformers` anyways why do we need to gate on `TYPE_CHECKING`?

Reply:
Modified this to remove the import. The real solution is to import of type checking AND try to import it when constructing the class/function but default to setting the function to None if we can't import.